### PR TITLE
Implement NewsArticleCover class

### PR DESCRIPTION
**Pull request by Game4all**
This PR implements the `NewsArticleCover` class.
Note that I'm using a dummy struct (see `ArticleInfo` class in`NewsArticleCover` ) as there is no API struct for news article info for now

Visual reference | This PR
----------------|--------
70392687-f6814300-1a25-11ea-9453-74008d2f0fc9 | image |

### CODE REVIEW

File: osu.Game/Overlays/News/NewsArticleCover.cs [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+                    },
+                    Font = OsuFont.GetFont(Typeface.Exo, 16, FontWeight.Bold),
+                    Text = "by " + info.Author
+                }
+            };
+
+            bg.OnLoadComplete += d => d.FadeIn(250, Easing.In);
+        }
+
+        //news article cover background
```
Comment (bdach) [inline osu.Game/Overlays/News/NewsArticleCover.cs]: Kind of a redundant comment (as is the next one)

File: osu.Game/Overlays/News/NewsArticleCover.cs [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        Colour = Color4.Black.Opacity(0.5f),
+                    },
+                    new OsuSpriteText
+                    {
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
+                        Font = OsuFont.GetFont(Typeface.Exo, 12, FontWeight.Black, false, false),
+                        Text = date.ToString("dd MMM yyy"),
```
Comment (bdach) [inline osu.Game/Overlays/News/NewsArticleCover.cs]: Don't know if we're after parity with web here, but if we are then this should probably be
```suggestion
                        Text = date.ToString("d MMM yyy").ToUpper(),
```

File: osu.Game/Overlays/NewsOverlay.cs [hunk 3]
```diff
@@ -59,6 +59,21 @@ private void load(OsuColour colours)
             Current.TriggerChange();
         }
 
+        private CancellationTokenSource loadChildCancellation;
+
+        protected void LoadAndShowChild(NewsContent newContent)
```
Comment (peppy) [inline osu.Game/Overlays/NewsOverlay.cs]: content, not child

**Review by peppy — APPROVED** [review]
seems fine for now

**Pull request merged**